### DISCUSSION TITLE
Tool to search by hotkey (spotlight like) & GUI

### DISCUSSION THREAD

**Discussion by tygore587** [Dashlane/dashlane-cli#189] [Ideas]
Hi Dashlane Team,

I'm coming from another password manager and switched to dashlane. 
It had a small program in the background where I can hit a hotkey to show a spotlight like search for saved accounts and press cmd + c afterwards to copy the password.

Is it something you would be okay with to get build around your cli?

Especially I see some api keys in `requestApi.ts` and wanted to ask if it is okay to do these type of projects open source before starting.

The same goes for a new UI around the cli. I really loved your desktop app and it would be cool to have one again.
It would be a bigger project but I just wanted to ask if this would be okay :)

**Comment by jboillot** [comment]
Hello! I'm not from Dashlane Team but, if you are using an Apple laptop, maybe this Alfred project could satisfy your needs? https://github.com/BenjaminOddou/alfred-dashlane

**Reply by tygore587** [reply]
Thanks for the suggestion. Sadly I'm using a combination of windows and mac.

**Comment by Mikescops** [comment]
Hello, 

Sorry for the late reply, we're revising our external contributor policy in the next days to clearly state what we allow or not. 

Concerning developing around our CLI, you're welcome to do so, we're supportive of anyone building new tools. We'll clarify all the details around our API usage and third party software distribution in our policy but this shouldn't prevent you from starting to develop your solution 👍
